script.js

```diff
@@ -1,5 +1,202 @@
+/* 
+There are 2 players and players take turns.
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.
+*/
+
+// Global Var
+player1diceResults = []
+player2diceResults = []
+player1 = ""
+player2 = ""
+player1Number = 1
+player2Number = 2
+// 
+
+
+var rollDice = function(){
+  // Create a dice rolling from 1 - 6
+  var randomDice = Math.floor(Math.random() * 6) + 1
+  console.log(`Dice roll: ${randomDice}`)
+  return randomDice
+}
+
+var roll2Dice = function(array){
+  firstDice = rollDice()
+  secondDice = rollDice()
+  array.push(firstDice)
+  array.push(secondDice)
+  console.log(`roll2Dice: ${firstDice}, ${secondDice}`)
+  console.log(`array: ${array}`)
+  console.log(`player1diceResults: ${player1diceResults}`)
+  console.log(`player2diceResults: ${player2diceResults}`)
+  return array
+}
```

Comment:
Again, as `firstDice` and `secondDice` are not global variables, do define them using `var`

Alternatively you can just do `array = [rollDice(), rollDice()]` and do away with the `firstDice` and `secondDice` variables altogether